server/src/test/java/io/spine/server/QueryServiceTest.java

```diff
@@ -77,9 +70,10 @@ void setUp() {
         boundedContexts.add(projectsContext);
 
         // Create Customers Bounded Context with one repository.
-        customersContext = BoundedContext.newBuilder()
-                                         .setName("Customers")
-                                         .build();
+        BoundedContext customersContext = BoundedContext
+                .newBuilder()
```

Comment:
Let's format this bounded context initialization expression in the same manner as the one a few lines above.